.rubocop_todo.yml

```diff
@@ -184,47 +184,25 @@ Style/MultilineMethodCallIndentation:
 Style/MultilineOperationIndentation:
   Enabled: false
 
-# Offense count: 2
-Style/NestedParenthesizedCalls:
-  Enabled: false
-
 # Offense count: 6
 # Cop supports --auto-correct.
 Style/ParallelAssignment:
   Enabled: false
 
-# Offense count: 32
-# Cop supports --auto-correct.
-Style/RedundantParentheses:
-  Enabled: false
-
 # Offense count: 8
 # Cop supports --auto-correct.
 # Configuration parameters: EnforcedStyle, SupportedStyles, AllowInnerSlashes.
 # SupportedStyles: slashes, percent_r, mixed
 Style/RegexpLiteral:
   Enabled: false
 
-# Offense count: 5
-# Cop supports --auto-correct.
-# Configuration parameters: EnforcedStyle, SupportedStyles.
-# SupportedStyles: space, no_space
-Style/SpaceInsideStringInterpolation:
```

Comment:
I wasn't too sure about this cop but since there were only 5 occurrences of spaces inside interpolation, I decided to enable this one for consistency.